lib/Music/music_album_manager.rb

```diff
@@ -9,19 +9,38 @@ def initialize
   end
 
   def list_music_albums
-    puts 'There is no albums yet.' if @albums.empty?
-    @albums.each_with_index do |album, index|
-      print "#{index + 1}) "
-      puts "Genre: \"#{album.genre.name.upcase}\", Published Date:#{album.publish_date}, On Spotify?: #{album.on_spotify ? 'Yes' : 'No'}"
+    puts "\e[37m"
```

Comment:
Hello @batoolfatima2135,
I can see that you've done a great job implementing the new lines for user-friendliness 😄👍. However, in this code, I was printing the albums with the number for each so that I could reuse this code in the `remove_album` method for selecting the number of the album to delete. Kindly add this code again.